lib/src/dart_formatter.dart

```diff
@@ -83,22 +84,7 @@ class DartFormatter {
   /// Returns a new [SourceCode] containing the formatted code and the resulting
   /// selection, if any.
   SourceCode formatSource(SourceCode source) {
-    // Enable all features that are enabled by default in the current analyzer
-    // version.
-    // TODO(paulberry): consider plumbing in experiment enable flags from the
-    // command line.
-    var featureSet = FeatureSet.fromEnableFlags2(
-      sdkLanguageVersion: Version(2, 19, 0),
-      flags: [
-        // TODO(rnystrom): This breaks existing switch cases containing constant
-        // expressions that aren't valid patterns. See:
-        // https://github.com/dart-lang/dart_style/issues/1164
-        'patterns',
-        'records',
-        'unnamed-libraries',
-      ],
-    );
-
+    // If we're parsing a single statement, wrap the source in a fake function.
```

Comment:
[nit] consider moving this comment to immediately above or below the `if` statement it relates to.

Reply:
Done.